src/layoutBuilder.js

```diff
@@ -661,12 +664,69 @@ LayoutBuilder.prototype.buildNextLine = function (textNode) {
 		return newInline;
 	}
 
+	function fixedCharCodeAt(str, idx) {
```

Comment:
Is there an appropriate location for these to be installed as utilities in `pdfmake`?